scripts/test.sh

```diff
@@ -2,16 +2,18 @@
 
 set -e
 
-if [[ $TRAVIS_SECURE_ENV_VARS  == 'true' ]]; then
-    yarn test
-    
-    # Continuous Deployment for staging/production
-    if [[ $TRAVIS_BRANCH == 'master' ]]; then
-        if [[ $TRAVIS_PULL_REQUEST == 'false' ]]; then
+yarn test
+
+# Continuous Deployment for staging/production
+if [[ $TRAVIS_BRANCH == 'master' ]]; then
+    if [[ $TRAVIS_PULL_REQUEST == 'false' ]]; then
+        if [[ ! -z $STATIC_PROD_PROJECT ]] && [[ ! -z $STATIC_DEPLOY_KEY ]]; then
```

Comment:
These checks are more granular and therefore preferred over the general `$TRAVIS_SECURE_ENV_VARS` check removed above.